aten/src/ATen/native/native_functions.yaml

```diff
@@ -8875,6 +8875,14 @@
   python_module: linalg
   variants: function
 
+- func: linalg_tensorinv(Tensor self, int ind=2) -> Tensor
+  python_module: linalg
+  variants: function
```

Comment:
Since the recent dispatch update, this needs to add a dispatch. Since it's composed of existing PyTorch operators I think it (and its out variant) get a "math" dispatch.

Reply:
The same answer as here https://github.com/pytorch/pytorch/pull/45358#discussion_r510118744

If there is no dispatch section, then "math" is assumed implicitly. Do we want it to be there explicitly?